storybookjs/storybook
# Rename `storyByModuleExport` to `resolveModuleExport`

So it can return either, allowing generic uses of `of={}`

Issue:

## What I did

Renamed the call, updated usages.

## How to test

- I think we should add unit tests for `docsContext`. We should potentially wait to merge this.
- Basic usages should be tested by e2e tests I think.
- We don't currently automatically test external docs so we should check that still works.
